File: iFlat/web/app/model/system/Role.js
Ext.define('iFlat.model.system.Role', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'role.roleId', mapping: 'roleId', type: 'string'},
        {name: 'role.roleName', mapping: 'roleName', type: 'string'},
        {name: 'role.category', mapping: 'category', type: 'string'},
        {name: 'role.description', mapping: 'description', type: 'string'},
        {name: 'role.status', mapping: 'status', type: 'boolean'},
    ]
});

File: src/main/java/com/iflat/system/bean/Role.java
package com.iflat.system.bean;

import java.util.Date;

/**
 * Created by tyriv on 2015/8/28.
 */
public class Role {

    //id
    private String roleId;
    //名称
    private String roleName;
    //分类
    private String category;
    //描述
    private String description;
    //1-启用 0-禁用
    private boolean status;
    //创建时间
    private Date createTime;

    public String getRoleId() {
        return roleId;
    }

    public void setRoleId(String roleId) {
        this.roleId = roleId;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getRoleName() {
        return roleName;
    }

    public void setRoleName(String roleName) {
        this.roleName = roleName;
    }

    public boolean getStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}
